src/App.jsx

```diff
@@ -8,17 +8,27 @@ import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
 import 'rheostat/initialize';
 import withTracker from './withTracker';
 
-export const App = () => (
-    <Router>
-        <main role="main">
-            <Route exact path="/" component={withTracker(Home)} />
-            <Route path="/central-limit-theorem"
-                component={withTracker(CentralLimitGraph)} />
-            <Route path="/ols-regression"
-                component={withTracker(OrdinaryLeastSquares)} />
-        </main>
-    </Router>
-);
+export class App extends Component {
+    constructor(props) {
```

Comment:
@nikolas is the needed here? I don't think it is because its the root component.

Reply:
It's not needed to override the constructor here, not because it's the root component, but because you're not doing any custom work in the constructor. 

When you don't include the constructor it just uses the parent's constructor, which just calls `super(props)`. It won't behave any differently, so just remove it here.

Reply:
And "parent" here refers to the class you're inheriting from: `Component`, not parent in the DOM tree.

Reply:
One more clarification: the parent doesn't call `super(props)`, rather, `super(props)` tells your component to run the parent's constructor, with the `props` parameter. React probably does some special things in its component constructor, that we don't necessarily need to know about. When overloading any constructor it's best practice to call `super()` first thing to make sure any setup the parent does is done.